packages/replay/src/replay.ts

```diff
@@ -854,3 +868,20 @@ export class ReplayContainer implements ReplayContainerInterface {
     return true;
   };
 }
+
+function _getExperimentalOptions(options: ReplayPluginOptions): ReplayExperimentalPluginOptions {
+  const requestHeaders = options._experiments.captureRequestHeaders || [];
+  const responseHeaders = options._experiments.captureResponseHeaders || [];
+  const captureBodies = options._experiments.captureNetworkBodies || false;
+
+  // Add defaults
+  const defaultHeaders = ['content-length', 'content-type', 'accept'];
+
+  return {
+    network: {
+      captureBodies,
+      requestHeaders: [...defaultHeaders, ...requestHeaders.map(header => header.toLowerCase())],
+      responseHeaders: [...defaultHeaders, ...responseHeaders.map(header => header.toLowerCase())],
```

Comment:
I'm just wondering (disregard if you already talked about this with the replay team) if there'd be a reason for users to actively disable sending the default headers?

Reply:
I think it's fine for now to always include these headers.

Reply:
We can always adjust this later IMHO and reduce the defaults, but I think these should be pretty safe for all use cases?